common/utils/domain.ts

```diff
@@ -1,8 +1,5 @@
 export function getDomain(url: string): string {
-  return url
-    .replace('http://', '')
-    .replace('https://', '')
-    .split('/')[0];
+  return url.replace('http://', '').replace('https://', '').split('/')[0];
```

Comment:
I think this is equivalent and neater?

```suggestion
  return (new URL(url)).hostname;
```